docs/config.md

```diff
@@ -75,20 +75,20 @@ Your kubernetes cluster must be reachable from wherever cluster-api-provider-ope
 ```bash
 openstack router create kube-router
 openstack router set kube-router --external-gateway public
-openstack router add subnet kube-nodes-subnet
+openstack router add subnet kube-router kube-nodes-subnet
 ```
 
 For another example on networking in openstack, look here https://developer.openstack.org/firstapp-libcloud/networking.html
 
 ## Security Group Rules
 
-For the installer to work, a few security groups are required to be open. These may be different from the security groups needed to reach a cluster once its running. The following security group rules should be added to the security group of your chosing. For this example, we will suppose you created a security group names ``kubernetes`` that you will use for the cluster.
+For the installer to work, a few security groups are required to be open. These may be different from the security groups needed to reach a cluster once its running. The following security group rules should be added to the security group of your chosing. For this example, we will suppose you created a security group named ``kubernetes`` that you will use for the cluster.
 
 ```bash
 openstack security group rule create --ingress --protocol tcp --dst-port 22 kubernetes
 openstack security group rule create --ingress --protocol tcp --dst-port 3000:32767 kubernetes
 openstack security group rule create --ingress --protocol tcp --dst-port 443 kubernetes
-openstack security group rule create --egress
+openstack security group rule create --egress kubernetes
```

Comment:
@jichenjc Wanted a feedback on this . As I mentioned in the description of this PR ,I did not get why there are no protocols for this egress rule.

Reply:
I am not sure, this was added by @iamemilio ,maybe we can get info from..

and if it's just 'kubenetes' missing, I assume that's a typo..., anyway, we can confirm from @iamemilio

Reply:
Okay I will wait for his reply and then push a patchset for the changes.

Reply:
I think this change is valid as network name is a positional argument in this command.
Will update if further chnage required